.github/workflows/consumer_contract_tests.yml

```diff
@@ -0,0 +1,100 @@
+name: Consumer contract tests
+# The purpose of this workflow is to validate the service level contract
+# using the Pact framework.
+#
+# More details on Contract Testing can be found in our handbook
+#
+# https://broadworkbench.atlassian.net/wiki/spaces/IRT/pages/2660368406/Getting+Started+with+Pact+Contract+Testing
+#
+#
+#
+# NOTE: The publish-contracts workflow will use the latest commit of the branch that triggers this workflow to publish the unique consumer contract version to Pact Broker.
+
+on:
+  pull_request:
+    branches: [ main ]
+    paths-ignore: [ '**.md' ]
+  push:
+    branches: [ main ]
+    paths-ignore: [ '**.md' ]
+  merge_group:
+    branches: [ main ]
+    paths-ignore: [ '**.md' ]
+
+jobs:
+  init-github-context:
+    runs-on: ubuntu-latest
+    outputs:
+      repo-branch: ${{ steps.extract-branch.outputs.repo-branch }}
```

Comment:
We also need to add `repo-version` to the `outputs` parameter here. Follow the example in [https://github.com/DataBiosphere/leonardo/blob/develop/.github/workflows/consumer_contract_tests.yml#L53](https://github.com/DataBiosphere/leonardo/blob/develop/.github/workflows/consumer_contract_tests.yml#L53)